open-hackathon-client/src/client/config_sample.py

```diff
@@ -258,7 +259,8 @@
                 "talent": {
                     "list": ["get"]
                 },
-                "grantedawards":["get"]
+                "grantedawards":["get"],
+                "islocal": ["get"]
```

Comment:
Why add an extra API? You can read the value from `config.py`. We have a `environment` configuratioin in both open hackathon client and server